docs/reference/logging/etwlogs.md

```diff
@@ -0,0 +1,74 @@
+<!--[metadata]>
++++
+title = "ETWlogs logging driver"
+description = "Describes how to use the etwlogs logging driver."
+keywords = ["ETW, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging" 
+weight=2
++++
+<![end-metadata]-->
+
+
+# ETWlogs logging driver
+
+The ETWlogs logging driver forwards container logs as ETW events. 
+ETW stands for Event Tracing in Windows, and is the common framework
+for tracing applications in Windows. Each ETW event contains a message
+with both the log and its context information. A client can then create
+an ETW listener to listen to these events. 
+
+The ETW provider that this logging driver registers with Windows, has the 
+GUID identifier of: a3693192-9ed6-46d2-a981-f8226c8363bd. A client creates an 
+ETW listener and registers to listen to events from the logging driver's provider. 
+It does not matter the order in which the provider and listener are created. 
+A client can create their ETW listener and start listening for events from the provider, 
+before the provider has been registered with the system. 
+
+## Usage
+
+Here is an example of how to listen to these events using the logman utility program 
+included in most installations of Windows:
+
+   1. logman start -ets DockerContainerLogs -p {a3693192-9ed6-46d2-a981-f8226c8363bd} 0 0 -o trace.etl
+   2. Run your container(s) with the etwlogs driver, by adding --log-driver=etwlogs 
+   to the Docker run command, and generate log messages.
+   3. logman stop -ets DockerContainerLogs
+   4. This will generate an etl file that contains the events. One way to convert this file into 
+   human-readable form is to run: tracerpt -y trace.etl. 
+   
+Each ETW event will contain a structured message string in this format:
+container_name: %s, image_name: %s, container_id: %s, image_id: %s, source: [stdout | stderr], log: %s
+
+Details on each item in the message can be found below:
+
+| Field                | Description                               |
+--------------------------------------------------------------------
+| container_name       | The container name at the time it was 
+|                      | started, minus the initial '/' character. |
+| image_name           | The name of the container's image.        |
+| container_id         | The full 64-character container ID.       |
+| image_id             | The full ID of the container's image.     |
+| source               | `stdout` or `stderr`.                     |
+| log                  | The container's log message.              |
+
+Here is an example string:
+container_name: backstabbing_spence, 
```

Comment:
If you want this to be a code block, there needs to be a newline and then indent four spaces.